qurator/eynollah/utils/pil_cv2.py

```diff
@@ -6,22 +6,23 @@
 # from sbb_binarization
 
 def cv2pil(img):
-    return Image.fromarray(img.astype('uint8'))
+    return Image.fromarray(img)
```

Comment:
Considering `pil2cv` and `cv2.imread` use BGR on the OpenCV side, shouldn't this function also do a `COLOR_BGR2RGB` conversion then?

Reply:
This is not used at the moment (the only caller is `check_dpi()`, which only gets called with PIL.Image or str, but not numpy.ndarray). But still, for the future, I think this should be fixed.

Reply:
Right, didn't see this comment before, sure, I'll change it.

Reply:
Was already fixed in 4897cefd

Reply:
> Right, didn't see this comment before, sure, I'll change it.

> Was already fixed in [4897cef](https://github.com/qurator-spk/eynollah/commit/4897cefdb70769128693d00b58f3148b5702eb71)

What do you mean, _already fixed_? `cv2pil` in the current tip of this branch still needs to do `COLOR_BGR2RGB` conversion AFAICT.

Reply:
I was under the impression that the proposed changeset was all that was missing. I'll add the conversion.